laravel/database/seeds/FilterRotationTableSeeder.php

```diff
@@ -11,22 +12,30 @@ class FilterRotationTableSeeder extends Seeder
      */
     public function run()
     {
-        if (file_exists(__DIR__ . "/../../../resources/database/SurgeonRotations.csv")) {
-            // Read data from the backup file and add into database
-            $fp = fopen(__DIR__ . "/../../../resources/database/SurgeonRotations.csv", "r");
+        // Make sure the file containing FilterRotation data exists. If it does not, print a warning and return.
+        if (!file_exists(__DIR__ . "/../../../resources/database/SurgeonRotations.csv")) {
+            print "WARNING: resources/database/SurgeonRotations.csv not found. Can't seed Filter Rotation Table." .
```

Comment:
Just curious, any reason why you are using `print` instead of using Logging, like `Log::debug`?

Reply:
No good reason. I'll change it to `Log::warning`.